# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/cookiebanners/metrics.yaml
# Commit: 3a37a1701348
# Full Hash: 3a37a1701348cc5dacf569187f586b0e11c3ed5e
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-03-29 04:51:52
# Regressor Bug: 1874741
# File Overlap Count: 1
# Description:
#   Bug 1874741 - Part 5: Implement the GDPR event telemetry for private browsing windows. r=pbz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205900
# ==============================================================================

diff -r 5f5dbfcfc95d -r 3a37a1701348 toolkit/components/cookiebanners/metrics.yaml
--- a/toolkit/components/cookiebanners/metrics.yaml	Thu Mar 28 20:46:34 2024 +0000
+++ b/toolkit/components/cookiebanners/metrics.yaml	Thu Mar 28 20:46:35 2024 +0000
@@ -255,6 +255,26 @@
           The region where the GDPR consent is made. This is based on the IP
           location.
         type: string
+  google_gdpr_choice_cookie_event_pbm:
+    type: event
+    description: >
+      Recorded whenever a GDPR choice is made on a Google Search page on private
+      browsing windows. We assess the "SOCS" cookie to know the GDPR choice.
+    bugs:
+      - https://bugzilla.mozilla.org/1874741
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1874741#c10
+    data_sensitivity:
+      - stored_content
+    notification_emails:
+      - pbz@mozilla.com
+      - tihuang@mozilla.com
+    expires: never
+    extra_keys:
+      choice:
+        description: >
+          The GDPR choice. The value could be "Accept", "Reject" or "Custom".
+        type: string
 
 cookie.banners.click:
   handle_duration: